src/main/resources/api/swagger.yaml

```diff
@@ -12,104 +12,103 @@ produces:
 paths:
   /dependencies:
     get:
-      summary: Topologicaly sorted list of features.
-      description: |
-        Returns the list of features topologicaly sorted base on the dependencies base on list of top features.
+      description: Returns the list of features topologically sorted based on the dependencies.
       tags:
         - Features
       parameters:
         - name: names
           in: query
-          description: Exact names of the features the call should return.
+          description: The exact names of the features the call should return.
           required: true
           type: array
           maxItems: 128
           items:
             type: string
       responses:
         200:
-          description: A list of features.
+          description: The request was successful.
           schema:
             $ref: '#/definitions/DependenciesList'
         default:
-          description: Unexpected error.
+          description: An unexpected error occurred.
           schema:
             $ref: '#/definitions/Error'
   /features:
     get:
-      summary: Get features.
-      description: Returns the features sorted by name.
+      description: Returns all features sorted by name.
       tags:
         - Features
       parameters:
         - name: q
           in: query
-          description: Part of the name of the features the call should return.
+          description: A part of the name of features to return.
           required: false
           type: string
         - name: author
           in: query
-          description: Name of the author of the returned features.
+          description: The name of the author of features to return.
           required: false
           type: string
         - name: fields
           in: query
-          description: The fields present in the response (JSON Field).
+          description: The JSON fields present in the response.
           required: false
           type: string
         - name: status
           in: query
-          description: Status of the features the call should return.
+          description: The status of features to return.
           required: false
           default: approved
           type: string
           enum:
-            - approved
-            - pending
-            - declined
+          - approved
+          - pending
+          - declined
```

Comment:
I checked the specs before, here an excerpt from http://swagger.io/specification/

```
Cat:
    description: A representation of a cat
    allOf:
    - $ref: '#/definitions/Pet'
    - type: object
      properties:
        huntingSkill:
          type: string
          description: The measured skill for hunting
          default: lazy
          enum:
          - clueless
          - lazy
          - adventurous
          - aggressive
      required:
      - huntingSkill
```